services/session.go

```diff
@@ -315,7 +318,10 @@ func LoadSessionsFromDisk() error {
 			}
 
 			// Connect PWD daemon to the new network
-			if err := ConnectNetwork("pwd", s.Id); err != nil {
+			if s.PwdIpAddress == "" {
+				log.Fatal("Cannot load stored sessions as they don't have the pwd ip address stored with them")
```

Comment:
Yes. This is a safeguard. If it fails to load it's because an old version of the saved sessions file is used and should be removed.
This way we don't have mixed sessions.